serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPlugin.scala

```diff
@@ -10,17 +10,23 @@ class SerializabilityCheckerCompilerPlugin(override val global: Global) extends
     """checks whether a specified Akka serialization is applied to all messages, events and persistent state classes"""
 
   //Placeholder options
-  private val pluginOptions = new SerializabilityCheckerOptions(verbose = false)
```

Comment:
While you're here... pls make sure that the plugin names listed in scalac-plugin.xml are consistent with `name` for both this compiler plugin and the dump schema plugin

Reply:
Not sure tho if those names in scalac-plugin.xml matter for anything... AFAICS when passing options via `scalacOptions`, it's `def name` that matters, not `<name>`